packages/expo-router/src/views/Sitemap.tsx

```diff
@@ -59,13 +60,23 @@ export function getNavOptions(): NativeStackNavigationOptions {
 }
 
 export function Sitemap() {
+  // Following the https://github.com/expo/expo/blob/ubax/router/move-404-and-sitemap-to-root/packages/expo-router/src/getRoutesSSR.ts#L38
+  // we need to ensure that the Sitemap component is not rendered on the server.
+  return (
+    <NoSSR>
+      <SitemapInner />
+    </NoSSR>
+  );
+}
+
+function SitemapInner() {
   const sitemap = useSitemap();
   const children = React.useMemo(
     () => sitemap?.children.filter(({ isInternal }) => !isInternal) ?? [],
     [sitemap]
   );
   return (
-    <View style={styles.container}>
+    <View style={styles.container} testID="expo-router-sitemap">
       {canOverrideStatusBarBehavior && <StatusBar barStyle="light-content" />}
       <ScrollView contentContainerStyle={styles.scroll}>
```

Comment:
```suggestion
      <ScrollView contentContainerStyle={styles.scroll} contentInsetAdjustmentBehavior="automatic">
```
Since there's no stack header, we need to account for safe area. Not sure if this works on Android though.

<img width="2732" height="2048" alt="Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) (16GB) - 2025-08-01 at 12 51 33" src="https://github.com/user-attachments/assets/91eba4f8-c6a8-4f27-b86a-7f3b39fed122" />